runtime-modules/membership/src/tests/mod.rs

```diff
@@ -1378,6 +1383,47 @@ fn successful_member_remark() {
             RawOrigin::Signed(ALICE_ACCOUNT_ID).into(),
             ALICE_MEMBER_ID,
             msg,
+            Some((BOB_ACCOUNT_ID, ed())),
+        );
+
+        assert_eq!(
+            validation_result,
+            Err(Error::<Test>::InsufficientBalanceToCoverPayment.into())
+        );
+    });
+}
+
+#[test]
+fn successful_member_remark_without_payment() {
+    build_test_externalities_with_lead_set().execute_with(|| {
+        set_alice_as_initial_member();
+
+        let msg = b"test".to_vec();
+        let validation_result = Membership::member_remark(
+            RawOrigin::Signed(ALICE_ACCOUNT_ID).into(),
+            ALICE_MEMBER_ID,
+            msg,
+            None,
+        );
+
+        assert_eq!(validation_result, Ok(()),);
+    });
+}
+
+#[test]
+fn successful_member_remark_with_payment() {
+    build_test_externalities_with_lead_set().execute_with(|| {
+        const ALICE_FREE_BALANCE: u64 = 100;
```

Comment:
- `ALICE.free_balance = 100` => `ALICE.usable_balance = 100` since no locks are enforced here
- the payment check in the *-remark extrinsic requires that `ALICE.usable_balance >= amount_to_pay  + ExistentialDeposit::get()`
- in the invocation you are setting `amount_to_pay = ALICE.usable_balance` and `ExistentialDeposit = 10` (in the mock configuration)

Is this test passing?